src/generators/minecraft-character/MinecraftCharacterGenerator.res

```diff
@@ -170,24 +100,80 @@ let script = () => {
     true,
   )
 
-  Generator.defineRegionInput((72, 24, 264, 198), () => {
-    Generator.setBooleanInputValue("Show Head Overlay", !showHeadOverlay)
-  })
-  Generator.defineRegionInput((266, 198, 198, 166), () => {
-    Generator.setBooleanInputValue("Show Body Overlay", !showBodyOverlay)
-  })
-  Generator.defineRegionInput((382, 372, 134, 166), () => {
-    Generator.setBooleanInputValue("Show Left Arm Overlay", !showLeftArmOverlay)
-  })
-  Generator.defineRegionInput((99, 372, 134, 166), () => {
-    Generator.setBooleanInputValue("Show Right Arm Overlay", !showRightArmOverlay)
-  })
-  Generator.defineRegionInput((383, 584, 134, 166), () => {
-    Generator.setBooleanInputValue("Show Left Leg Overlay", !showLeftLegOverlay)
-  })
-  Generator.defineRegionInput((99, 584, 134, 166), () => {
-    Generator.setBooleanInputValue("Show Right Leg Overlay", !showRightLegOverlay)
-  })
+  let char = isAlexModel ? alex : steve
+
+  let drawHead = ((ox, oy): Generator_Builder.position) => {
+    let scale = (64, 64, 64)
+    Minecraft.drawCuboid("Skin", char.base.head, (ox, oy), scale, ())
+    if showHeadOverlay {
+      Minecraft.drawCuboid("Skin", char.overlay.head, (ox, oy), scale, ())
+    }
+    /* if showFolds {
+      Generator.drawFoldLineCuboid((ox, oy), scale, ())
+    } */
+  }
+
+  let drawBody = ((ox, oy): Generator_Builder.position) => {
+    let scale = (64, 96, 32)
+    Minecraft.drawCuboid("Skin", char.base.body, (ox, oy), scale, ())
+    if showBodyOverlay {
+      Minecraft.drawCuboid("Skin", char.overlay.body, (ox, oy), scale, ())
+    }
+    /* if showFolds {
+      Generator.drawFoldLineCuboid((ox, oy), scale, ())
+    } */
+  }
+
+  let drawRightArm = ((ox, oy): Generator_Builder.position) => {
+    let scale = char == alex ? (24, 96, 32) : (32, 96, 32)
```

Comment:
The `char == alex` might be the cause of the glitch I mentioned in the other comment. Would triple equals fix it? I.e. `char === alex`?